### PR TITLE
Add TechPowerUp GPU-Z 0.6.6

### PR DESCRIPTION
**Pull request by jeditobe**
it works on recent nightly builds of ReactOS

A freeware tool that gathers information on the GPU devices of your system.

### CODE REVIEW

**Review by JoachimHenze — APPROVED** [review]

**Review by learn-more — APPROVED** [review]

File: gpuz.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Section]
+Name = TechPowerUp GPU-Z
```
Comment (JoachimHenze) [inline gpuz.txt:2]: Please strip the EOL whitespace at the end of line 2.
Afterwards I will merge the PR.

Comment (jeditobe) [reply gpuz.txt]: done

**Comment by JoachimHenze** [comment]
@jeditobe Your download link is "410 Gone" and can neither be accessed from Windows nor from ros. :-(
https://de1-dl.techpowerup.com/files/inMaEKiEApD-BpFnze5P3w/1675732482/GPU-Z.0.6.6.exe
Please update it with something working in a newer PR!

**Comment by learn-more** [comment]
That looks like one of those shitty sites that places a cookie for a download to work.

**Comment by jeditobe** [comment]
[JoachimHenze](https://github.com/JoachimHenze) Please don't treat me like a fool, I've checked the link availability many times during the week before posting. And the link worked on different computers, including the incognito tab of the browser

**Pull request merged**